docs/app/Examples/collections/Table/States/TableWarningShorthand.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Table } from 'stardust'
+
+const headerRow = [
+  'Name',
+  'Status',
+  'Notes',
+]
+
+const bodyRow = ({ name, status, notes }) => ({
+  warning: !!(status && status.match('Requires Action')),
+  items: [
+    name || 'No name specified',
+    status
+      ? { icon: 'attention', content: status }
+      : 'Unknown',
+    notes
+      ? { icon: 'attention', content: notes, warning: true }
+      : 'None',
+  ],
+})
+
+const tableData = [
+  { name: undefined, status: undefined, notes: undefined },
+  { name: 'Jimmy', status: 'Requires Action', notes: undefined },
+  { name: 'Jamie', status: undefined, notes: 'Hostile' },
+  { name: 'Jill', status: undefined, notes: undefined },
+]
+
+const TableWarningShorthand = () => {
+  return (
+    <Table celled headerRow={headerRow} bodyRow={bodyRow} tableData={tableData} />
```

Comment:
API suggestion:

```jsx
<Table 
  renderHeaderRow={...}
  renderBodyRow={...}
  renderFooterRow={...}
/>
```

Eventually, I'd like to also add column/cell shorthand.  These would disallow use of the row shorthand, but I think it would give the user the ability to map through their data in what ever way made the most sense to them.